docs/src/modules/components/MarkdownElement.js

```diff
@@ -71,21 +70,25 @@ const Root = styled('div')(({ theme }) => ({
   '& h2': {
     ...theme.typography.h5,
     fontFamily: theme.typography.fontFamilySystem,
+    fontSize: theme.typography.pxToRem(24),
     fontWeight: 700,
+    letterSpacing: '0px',
     color: theme.palette.mode === 'dark' ? theme.palette.grey[100] : theme.palette.grey[900],
-    margin: '40px 0 4px',
+    margin: '40px 0 8px',
```

Comment:
The style changes create a merge conflict. I think that we should revert them and move them to a different PR, one specific about improving the styles of the docs and blog pages.